demisto_sdk/commands/common/tools.py

```diff
@@ -161,6 +160,7 @@ def run_command(command, is_silenced=True, exit_on_error=True, cwd=None):
     return output
 
 
+@lru_cache(maxsize=8)
```

Comment:
I think you can increase this. 8 is quite small. You can put in 50.

Reply:
they recommended powers of 2. i'll do 64.

Reply:
they recommended powers of 2. i'll do 64.